lib/src/labels.js

```diff
@@ -0,0 +1,153 @@
+const addLabelsApi = matroid => {
+  // https://staging.dev.matroid.com/docs/api/index.html#api-Labels-PostDetectorsDetector_idLabels
+  matroid.createLabelWithImages = function(
+    detectorId,
+    name,
+    imageFiles,
+    configs = {}
+  ) {
+    /*
+    Create a label and upload images to it
+    Requires processing=false. Creates label asynchronously (turn processing to true)
+    */
+    return new Promise((resolve, reject) => {
+      this._checkRequiredParams({ detectorId, name, imageFiles });
+
+      let options = {
+        action: 'createLabelWithImages',
+        uriParams: { ':key': detectorId },
+        data: { name }
+      };
+
+      this._checkImageSize(imageFiles);
+      options.filePaths = { image_files: imageFiles };
+
+      let processedConfigs = configs;
+      if (processedConfigs.bboxes) {
+        processedConfigs.bboxes = JSON.stringify(processedConfigs.bboxes);
+      }
+
+      Object.assign(options.data, processedConfigs);
+
+      this._genericRequest(options, resolve, reject);
+    });
+  };
+
+  // https://staging.dev.matroid.com/docs/api/index.html#api-Labels-DeleteDetectorsDetector_idLabelsLabel_id
```

Comment:
Might want to change these `staging.dev.matroid.com` references to `matroid.com` since users who download this package will be able to read the source code.

Reply:
Fixed!